ios/RNCImageEditor.mm

```diff
@@ -57,7 +57,7 @@ - (void) cropImage:(NSString *)uri
   }
   NSString *displaySize = data.resizeMode();
   NSURLRequest *imageRequest = [NSURLRequest requestWithURL:[NSURL URLWithString: uri]];
-  CGFloat compressionQuality = 1;
+  CGFloat compressionQuality = 0.9;
```

Comment:
If the same default is used in 2 places (new or old arch), could you move to some constant?